pkg/alertmanager/multitenant.go

```diff
@@ -479,10 +483,16 @@ func (am *MultitenantAlertmanager) ServeHTTP(w http.ResponseWriter, req *http.Re
 	userAM, ok := am.alertmanagers[userID]
 	am.alertmanagersMtx.Unlock()
 
-	if !ok || !userAM.IsActive() {
-		http.Error(w, "no Alertmanager for this user ID", http.StatusNotFound)
+	if !ok {
+		http.Error(w, "no Alertmanager for this user ID, Alertmanager not configured", http.StatusNotFound)
```

Comment:
Given the request comes from the user itself, we may rephrase it?

```suggestion
		http.Error(w, "the Alertmanager has not been configured", http.StatusNotFound)
```

Reply:
I prefer the active voice instead of a passive one, but YMMV.

Reply:
Sure, feel free to phrase it as you think is better, my comment was about:
1. not mention "user ID" ("user ID" is an internal thing, the final user just cares about themselves)
2. "no Alertmanager for this user" looks a bit unclear to me, while "Alertmanager not configured" looks more clear, so we should just say the Alertmanager hasn't been configured yet